spec.md

```diff
@@ -2788,6 +2789,16 @@ They are interpreted according to Starlark's [indexing conventions](#indexing).
 "hello, world!".count("o", 7, 12)       # 1  (in "world")
 ```
 
+<a id='string·elems'></a>
+### string·elems
+
+`S.elems()` returns a list containing successive
+1-byte substrings of S.
```

Comment:
Replace list with iterable value.

Reply:
But the Java implementation returns a list

Reply:
And the Go implementation doesn't. Since lists are iterable, it's correct for both implementations to say we return an iterable.